File: 6kyu/squared_spiral_2/javascript/example_test_cases.js
const chai = require("chai");
const assert = chai.assert;

describe("Tests", function () {
  it("Fixed Tests", function () {
    assert.strictEqual(squaredSpiral(0, 0), 0);
    assert.strictEqual(squaredSpiral(1, 1), 2);
    assert.strictEqual(squaredSpiral(2, 0), 10);
    assert.strictEqual(squaredSpiral(2, 2), 12);
    assert.strictEqual(squaredSpiral(3, 3), 30);
  });
});

File: 6kyu/squared_spiral_2/c++/example_test_cases.cpp
Describe(tests)
{
    It(fixedTests)
    {
        Assert::That(squaredSpiral(0, 0), Equals(0));
        Assert::That(squaredSpiral(1, 1), Equals(2));
        Assert::That(squaredSpiral(2, 0), Equals(10));
        Assert::That(squaredSpiral(2, 2), Equals(12));
        Assert::That(squaredSpiral(3, 3), Equals(30));
    }
};